packages/childchain/README.md

```diff
@@ -1,14 +1,39 @@
 # Running the node
 
-To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed: 
+To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed:
 
 ```
 git clone https://github.com/ChainSafe/ethermint
 cd ethermint
 make install
 ```
-To run the chain run `./childchain.sh` and in a second terminal/tab `ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key mykey --chain-id 8 --trace`
 
 # Run the tests
 
 To compile the contracts run `truffle build`, to run the tests run `truffle test`.
```

Comment:
Do I need to do this before of after running start up script?

Reply:
Unresolving. `truffle test` will fail if server is not running so ether this part should be moved somewhere or the requirement of running ethermint should be added.